### PR TITLE
Rnd QoL Sorting and new node!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Makes a new illegal node! NV goggles for the hardsuit and eyes! Wow Sci!
Updates the Dquel hatter to use a Dquel hatter sprite
Makes all medical items into its own category
Makes all bluespace snar into the bluespace category 
New glowstick- Undark!
Lasts longer, and is brighter then leading brands!
## Why It's Good For The Game

The new node gives just a little more to get/give out for wealth. Could be nice and all for sci to work to it after getting illegal tech and be all like `Pss kid want some undark sight?`

QoL for sorting cuz its a pain to go into "unsorted" every 10 seconds to print anything I want medial wise.

Better sprite for a node that no one really gets/prints

## Changelog
:cl:
add: New node! NV goggles for hardsuits and normal eye ware.
add: New sprite for the D-qel Hatter, its the D-qel hatter spite. Shocked we know
code: The sorting no one asked for but everyone wants* - New medical tab for Rnd tech that has medical items. Bluespace Snar stuff now is in Bluespace tab.
add: New glowstick type, undark! Its better then the leading brands
/:cl:

### CODE REVIEW

**Comment by Kurgis** [comment]
It is odd that the "Illegal" node is still in use, considering nothing is illegal in this setting. Feels like a relic of Nanotrasen terminology.

File: code/modules/research/nodes/illegal.dm [hunk 1]
```diff
@@ -13,6 +13,22 @@
 
 	unlocks_designs = list(/datum/design/research/item/binaryencrypt)
 
+/datum/technology/night_sight
+	name = "Undark Vision"
```
Comment (MadmanMartian) [inline code/modules/research/nodes/illegal.dm:17]: >Undark vision
>!dark vision
>light vision

Call it digital vision enhancement or something

**Comment by MadmanMartian** [comment]
Maybe it should be renamed to covert node or something?

**Comment by Trilbyspaceclone** [comment]
The undark is a ref to a paint that used uranium in its paint to glow in the dark
https://en.wikipedia.org/wiki/Undark

**Comment by Trilbyspaceclone** [comment]
@Kurgis Fun fact we also use Arcane tech but!
But only one admin debug item gives the tech for it...

**Comment by MadmanMartian** [comment]
Well that's cool and 1920s as heck, would make sense for glowsticks or super flashlights

**Comment by Trilbyspaceclone** [comment]
I could prob throw in glowsticks if you want

**Pull request merged**